src/command.c

```diff
@@ -878,10 +878,10 @@ cmd_volume(int argc, char **argv, struct mpd_connection *conn)
 		struct mpd_status *status = getStatus(conn);
 
 		if (mpd_status_get_volume(status) >= 0)
-			printf("volume:%3i%c\n",
+			printf("volume: %3i%c\n",
```

Comment:
This is just another way of being inconsistent: instead of having zero, one or two spaces, now it's one, two or three spaces.